app/src/main/java/be/scri/fragments/MainFragment.kt

```diff
@@ -39,9 +42,34 @@ class MainFragment : ScribeFragment("Main") {
             }
         (requireActivity() as MainActivity).setActionBarButtonVisibility(false)
         callback.isEnabled = true
+        isArabicLanguage()
         return binding.root
     }
 
+    private fun isArabicLanguage(): Boolean {
+        val cornerTriangleImageView: ImageView = binding.cornerTriangle
```

Comment:
what about checking layout direction, so we would cover all RTL languages without need to explicitly check each language.

Reply:
If this is possible, then yes, but we can also just save a local variable for RTL languages that we support as as of now the plan is just Arabic and eventually Hebrew.